src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const mediumTermDiscount = 20;
+  const longTermDiscount = 50;
+  const shortTermRent = days * pricePerDay;
+  const mediumTermRent = days * pricePerDay - mediumTermDiscount;
+  const longTermRent = days * pricePerDay - longTermDiscount;
+
+  if (days < 3) {
```

Comment:
2. [CHECKLIST] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values.

Please, add changes for all code lines

Reply:
thank you! I'm definitely fixing that